src/lib.rs

```diff
@@ -92,14 +85,48 @@ pub enum TeePubKey {
     },
 }
 
+/// Data generated during the attestation process between client and server. Relevant only to the
+/// client-server pairing.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RuntimeData {
+    /// Nonce string generated by server.
+    pub nonce: String,
```

Comment:
this is set by KBS with the nonce that `Challenge` provided. I wonder if Deserialize should skip this?

Reply:
I was actually wondering why this was a field in the first place, since the KBS already stores it. Are you saying that instead of deserializing it from the RuntimeData, the attestation service instead should get it from the KBS?